Content.Shared/Whitelist/EntityWhitelistSystem.cs

```diff
@@ -113,6 +114,20 @@ public bool IsWhitelistPass(EntityWhitelist? whitelist, EntityUid uid)
         if (whitelist == null)
             return false;
 
+        // Begin DeltaV
+        Log.Debug($"Checking whitelist pass for entity {ToPrettyString(uid)}");
```

Comment:
dedundant because of the log right after it

Reply:
real